manual-install/update-yaml.sh

```diff
@@ -21,7 +21,7 @@ OUTPUT="$(echo "$OUTPUT" | jq 'del(.services[] | select(.container_name == "next
 OUTPUT="$(echo "$OUTPUT" | jq 'del(.services[] | select(.container_name == "nextcloud-aio-domaincheck"))')"
 OUTPUT="$(echo "$OUTPUT" | jq 'del(.services[] | select(.container_name == "nextcloud-aio-borgbackup"))')"
 OUTPUT="$(echo "$OUTPUT" | jq 'del(.services[] | select(.container_name == "nextcloud-aio-docker-socket-proxy"))')"
-OUTPUT="$(echo "$OUTPUT" | jq '.services[].depends_on |= if contains(["nextcloud-aio-docker-socket-proxy"]) then del(.[index("nextcloud-aio-docker-socket-proxy")])')"
+OUTPUT="$(echo "$OUTPUT" | jq '.services[].depends_on |=  if contains(["nextcloud-aio-docker-socket-proxy"]) then del(.[index("nextcloud-aio-docker-socket-proxy")])')"
```

Comment:
@Zoey2936 do you know how to fix this logic? This somehow still does not seem to work...

Reply:
I think there is a "else ." missing

Reply:
Oh it requires an else? 🤦‍♂️

Reply:
yes

Reply:
and an "end", so it needs to end with "else . end'"

Reply:
let me try this. thanks!

Reply:
Still seems to fail :/

![image](https://github.com/nextcloud/all-in-one/assets/42591237/63b0e117-a3b1-4c49-95be-b5a647b5face)

Reply:
I need to probably first check if depends_on is there and then check if it contains something?